### ISSUE TITLE
Choosing best model from all the checkpoints

### ISSUE THREAD

**Issue by nobodykid** [junyanz/pytorch-CycleGAN-and-pix2pix#690]
Hello, I've experimented a lot using pix2pix on different case and also using various modification based from your original paper. But I have a bit difficulty in deciding which checkpoint should I choose as "the best"

After reading your paper, it's still vague for me on how do you choose best model across numerous checkpoints that you saved during single run of training. Do you have strategy or certain validation method during training to see if the model had learned well?

**Comment by junyanz** [comment]
You can use the validation set for pix2pix models.

**Comment by nobodykid** [comment]
Validation set, as in 
splitting the data into train-val-test, then using the val set on certain iteration steps to see samples from the model?

**Comment by junyanz** [comment]
Yeah.

**Comment by nobodykid** [comment]
Aaah I see, so then I could to evaluation on those data generated using validation set.

Thank you for the advice
